beacon_node/beacon_chain/src/metrics.rs

```diff
@@ -1645,6 +1645,12 @@ pub static BLOB_SIDECAR_INCLUSION_PROOF_COMPUTATION: LazyLock<Result<Histogram>>
             "Time taken to compute blob sidecar inclusion proof",
         )
     });
+pub static DATA_COLUMN_SIDECAR_COMPUTATION: LazyLock<Result<Histogram>> = LazyLock::new(|| {
+    try_create_histogram(
+        "data_column_sidecar_computation_seconds",
+        "Time taken to compute data column sidecar, including cells, proofs and inclusion proof",
+    )
```

Comment:
Are the default bucket values good enough?

Reply:
Definitely not! (we probably overuse the default bucket too much)
I've udpated in https://github.com/sigp/lighthouse/pull/6212/commits/2fdcc356731d76c4fe3faf083e1b2347f54575a 6 based on some local benchmarks. Let me know what you think.

Reply:
Looks good!